src/geode/mesh/helpers/rasterize.cpp

```diff
@@ -440,17 +443,76 @@ namespace
         return cells;
     }
 
+    absl::InlinedVector< geode::GridCellIndices3D, 6 > neighbors(
+        const geode::RegularGrid3D& grid, const geode::GridCellIndices3D& cell )
+    {
+        absl::InlinedVector< geode::GridCellIndices3D, 6 > neighbors;
+        for( const auto d : geode::LRange{ 3 } )
+        {
+            if( const auto prev = grid.previous_cell( cell, d ) )
+            {
+                neighbors.push_back( prev.value() );
+            }
+            if( const auto next = grid.next_cell( cell, d ) )
+            {
+                neighbors.push_back( next.value() );
+            }
+        }
+        return neighbors;
+    }
+
     std::vector< geode::GridCellIndices3D > conservative_voxelization_segment(
         const geode::RegularGrid3D& grid,
         const geode::Segment3D& segment,
-        const std::array< geode::GridCellsAroundVertex3D, 2 > vertex_cells )
+        const std::array< geode::GridCellsAroundVertex3D, 2 > /*unused*/ )
     {
-        std::vector< geode::GridCellIndices3D > cells;
-        geode_unused( grid );
-        geode_unused( segment );
-        geode_unused( vertex_cells );
-        OPENGEODE_EXCEPTION( false,
-            "[conservative_voxelization_segment] Not implement yet for 3D" );
+        constexpr auto cell_tested_attribute_name = "tested_cell_rasterize";
+        auto cells = geode::rasterize_segment( grid, segment );
+        auto tested_att =
```

Comment:
Use a vector instead. With an attribute we cannot rasterize several segments in parallel